usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -54,6 +54,17 @@
         " tas_components ARRAY<STRING>",
         "column_names": list(AWARD_SEARCH_COLUMNS),
     },
+    "sam_recipient": {
```

Comment:
Ben, can you make the key for this table `int.sam_recipient`, so we won't have a key conflict with the TABLE_SPEC in `load_table_to_delta`?

Reply:
updated key to avoid conflict